network/forks/ordered.go

```diff
@@ -100,3 +100,26 @@ func NewOrderedSchedule(b *params.BeaconChainConfig) OrderedSchedule {
 	sort.Sort(ofs)
 	return ofs
 }
+
+// ForkForEpochFromConfig returns the fork data for the given epoch from the provided config.
+func ForkForEpochFromConfig(cfg *params.BeaconChainConfig, epoch primitives.Epoch) (*ethpb.Fork, error) {
```

Comment:
nice i think there are some other places that could use this helper in the future, need to double check some function in prysmctl

Reply:
I'm going to do another pass on this in a subsequent PR, because the way it works can still be improved.